sos/policies/redhat.py

```diff
@@ -281,6 +287,61 @@ def check(cls):
                         return True
         return False
 
+    def get_upload_url(self):
+        if self.case_id and not self.commons['cmdlineopts'].upload_url:
+            # in this case we want to default to using the RHCP for uploads so
+            # clear the default FTP user so that sos users will be prompted
+            # for an RHCP username
```

Comment:
I suggest rephrasing to something like "clear the default FTP use so that RHCP username must be provided via config / cmdline option or prompt". (current text confusingly ignores `--upload-user` option)

Reply:
Ack.